Assets/_Functionalities/ObjectInformation/Scripts/BagInspector.cs

```diff
@@ -145,10 +146,21 @@ private void FindObjectMapping()
                 //lets order all mappings by layerorder (rootindex) from layerdata
                 if (featureSelector.HasFeatureMapping)
 				{
+					List<Feature> filterDuplicates = new List<Feature>();
 					foreach (FeatureMapping feature in featureSelector.FeatureMappings)
 					{
-						if(feature.VisualisationParent.LayerData.ActiveInHierarchy)
+						if (feature.VisualisationParent.LayerData.ActiveInHierarchy)
+						{
+							if(filterDuplicateFeatures)
+							{
+								if (!filterDuplicates.Contains(feature.Feature))
+									filterDuplicates.Add(feature.Feature);
+								else
+									continue;
```

Comment:
dus als feature al wel in de duplicates zit, dan moet die ook niet aan mappings worden toegevoegd? om dat leesbaarder te maken zou ik die mappings.TryAdd van hieronder in die if toevoegen en de else continue; weghalen